wandb/sdk/wandb_run.py

```diff
@@ -587,34 +605,40 @@ def __setstate__(self, state: Any) -> None:
 
         self._attach_id = _attach_id
 
-    @property
+    @property  # type: ignore
```

Comment:
this is an issue with property see here: https://github.com/python/mypy/issues/1362